src/ui/pages/Menu/Menu.tsx

```diff
@@ -57,100 +99,76 @@ const Menu = () => {
     );
   };
 
-  const handleItemSelection = (option: string) => {
-    switch (option) {
-    case i18n.t("menu.tab.items.lock"):
-      dispatch(logout());
-      return;
-    default:
-      return;
-    }
-  };
+  const menuItems = [
+    {
+      itemKey: SubMenuKey.Profile,
+      icon: personCircleOutline,
+      label: `${i18n.t("menu.tab.items.profile")}`,
+    },
+    {
+      itemKey: SubMenuKey.Crypto,
+      icon: walletOutline,
+      label: `${i18n.t("menu.tab.items.crypto")}`,
+    },
+    {
+      itemKey: SubMenuKey.Connections,
+      icon: peopleOutline,
+      label: `${i18n.t("menu.tab.items.connections")}`,
+    },
+    {
+      itemKey: SubMenuKey.P2P,
+      icon: chatbubbleOutline,
+      label: `${i18n.t("menu.tab.items.p2p")}`,
+    },
+    {
+      itemKey: SubMenuKey.Identifier,
+      icon: fingerPrintOutline,
+      label: `${i18n.t("menu.tab.items.identity")}`,
+    },
+    {
+      itemKey: SubMenuKey.Credential,
+      icon: idCardOutline,
+      label: `${i18n.t("menu.tab.items.credentials")}`,
+    },
+  ];
 
-  const MenuItem = ({
-    index,
-    icon,
-    label,
-  }: {
-    index: number;
-    icon: string;
-    label: string;
-  }) => {
-    return (
-      <IonCol>
-        <IonCard
-          onClick={() => handleItemSelection(label)}
-          data-testid={`menu-input-item-${index}`}
-          className="menu-input"
-        >
-          <IonIcon
-            icon={icon}
-            color="primary"
-          />
-          <IonLabel>{label}</IonLabel>
-        </IonCard>
-      </IonCol>
-    );
-  };
+  const selectSubmenu = useMemo(() => {
```

Comment:
There was a comment here from Duke that you've removed - I asked him to add this comment so it was clear that we need the `emptySubMenu` so that the animation won't be messed up. I think this is useful for maintainability.